trieste/bayesian_optimizer.py

```diff
@@ -545,7 +555,8 @@ def optimize(
         *,
         track_state: bool = True,
         track_path: Optional[Path | str] = None,
-        fit_model: str = "all",
+        fit_model: bool = True,
+        fit_initial_model: bool = True,
```

Comment:
Was the agreement to add multiple `fit*` arguments instead of a single one?

Reply:
The TuRBO PR replaced the existing `fit_initial_model: bool` argument with a `fit_model: str` argument that covers all three use cases. This works, but is a breaking change. Furthermore, having two boolean arguments is actually simpler and less error prone.